app/src/main/java/com/peacecorps/pcsa/safety_tools/CommunityFragment.java

```diff
@@ -0,0 +1,77 @@
+package com.peacecorps.pcsa.safety_tools;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.text.Html;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import com.peacecorps.pcsa.R;
+
+/*
+ * Community fragment, member of the Tabbed Layout
```

Comment:
"A member of the Tabbed Layout" just means that it is one of the subsections in the TabLayout I have created for Safety Plan